scripts/bin/update-packages

```diff
@@ -176,20 +198,22 @@ _fetch_and_cache_tags() {
 		termux_pkg_auto_update
 	}
 
+	# This function generates a ninja file for all packages in the ${__PACKAGES[@]}
 	__generate__() {
 		echo "rule update"
-		echo " command = bash -c \"\$\$F\" -- \$pkg_dir ||:"
+		echo " command = bash -c \"\$\$F\" -- \$pkg_dir || :"
 		sed 's/[^ ]\+/\nbuild &: update\n pkg_dir=&/g' <<< "${__PACKAGES[@]}"
-		echo "build run_all: phony ${__PACKAGES[@]}"
+		echo "build run_all: phony ${__PACKAGES[*]}"
 		echo "default run_all"
 	}
-	
-	local LATEST_TAGS="$(\
+
+	local LATEST_TAGS
+	LATEST_TAGS="$(
 		F="$(declare -p TERMUX_SCRIPTDIR GITHUB_TOKEN TERMUX_REPOLOGY_DATA_FILE); $(declare -f __main__ | sed 's/__main__ ()//')" \
-		env --chdir=/tmp  ninja -f /dev/stdin <<< "$(__generate__)" |& grep "^PKG|"
+		env --chdir=/tmp ninja -f /dev/stdin < <(__generate__ |& grep "^PKG|")
```

Comment:
grep must be applied to ninja, not to __generate__.

Reply:
Yep noticed that already locally just haven't pushed it yet.